### ISSUE TITLE
Kadane's Algorithm in Dynamic Programming Folder 

### ISSUE THREAD

**Issue by shivanithorve01** [Lakhankumawat/LearnCPP#49]
Kadane's algorithm is an iterative dynamic programming algorithm in which we search for a maximum sum contiguous subarray within a one-dimensional numeric array. 
@Lakhankumawat will you please assign this issue to me ?
I'm GSSoC 2022 Participant.

**Comment by Lakhankumawat** [comment]
Before 1st March you can contribute but points will be alloted only after 1st March

**Comment by shivanithorve01** [comment]
Okay !! Got It !!

**Comment by Lakhankumawat** [comment]
For more please join meet on 28 Feb as stated in repo's README

**Comment by shubhamdugar20** [comment]
Hey! I am a GSOC'22 participant . can u assign me this issue?

**Comment by Lakhankumawat** [comment]
Hi @shubhamdugar20  this issue is already assigned to @shivanithorve01  you can find another issue or can create your own issue , thanks for your interest .

**Comment by Lakhankumawat** [comment]
If you're not sure about issues then join the meet tomorrow as stated in README of the Repo
